api/agentpb/collector.proto

```diff
@@ -0,0 +1,325 @@
+syntax = "proto3";
+
+package agent;
+
+option go_package = "agentpb";
+
+import "inventorypb/agents.proto";
+import "qanpb/qan.proto";
+
+// MetricsBucket is aggregated message created by pmm-agent.
+// Contains information about one query selected in defined way from query class in specific period of time.
+message MetricsBucket {
+  // Common contains common fields for all DBs.
+  message Common {
+    // md5 of digest_text/fingerprint.
+    string queryid = 1;
+    // digest_text - query signature. Query without values.
+    string fingerprint = 2;
+    //
+    // Dimension Group.
+    //
+    // Service Name (can be several Services on one Node).
+    string service_name = 3;
+    // PostgreSQL database.
+    string database = 4;
+    // MySQL database or PostgreSQL schema.
+    string schema = 5;
+    // client user name.
+    string username = 6;
+    // client IP or hostname.
+    string client_host = 7;
+    //
+    // Unified Labels Group.
+    //
+    // Name of replication set
+    string replication_set = 10;
+    // Cluster name
+    string cluster = 11;
+    // Type of service
+    string service_type = 12;
+    // Environment name
+    string environment = 13;
+    // Availability zone
+    string az = 14;
+    // Region name
+    string region = 15;
+    // Node model
+    string node_model = 16;
```

Comment:
Many of those fields are set by pmm-managed, not by pmm-agent, so we can remove them from that half. That's the whole point of this task.